SampleProgram/step5.c

```diff
@@ -0,0 +1,19 @@
+#include <fcntl.h>
+#include <stdio.h>
+
+void main(void)
+{
+    int buff_size = 256;
+    FILE *fp;
+
+    char buff[buff_size];
+    while(1){
+        if ((fp = fopen("/dev/rtlightsensor0", "r")) != NULL){
+	    while(fgets(buff, buff_size, fp) != NULL){
+            	printf("%s\n", buff);
+	    }
```

Comment:
スペースとタブが混ざっているようです。ほかがスペースでインデントされているのでスペースに統一するのが望ましいと思います。

```suggestion
            while(fgets(buff, buff_size, fp) != NULL){
                printf("%s\n", buff);
            }
```

![Screenshot from 2020-04-03 19-31-39](https://user-images.githubusercontent.com/3256629/78351713-34603380-75e2-11ea-9b8c-03d090c35cd1.png)